src/main/java/com/google/codeu/data/Message.java

```diff
@@ -19,41 +19,56 @@
 import java.util.UUID;
 
 /** A single message posted by a user. */
-public class Message {
+public class Message
+{
 
   private UUID id;
   private String user;
   private String text;
   private long timestamp;
+  private String recipient; //added recipient for messages
 
   /**
    * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
    * random ID and uses the current system time for the creation time.
    */
-  public Message(String user, String text) {
-    this(UUID.randomUUID(), user, text, System.currentTimeMillis());
+  public Message(String user, String text, String recipient)/*Added recipient to list of constructors*/
```

Comment:
Refrain from commenting in line. Comments are usually reserved their own line. Also refrain from writing comments that someone reading your code can clearly see what the code is doing. Notice that parts of original code that do and do not have comments. It is important to follow what the original code does and make sure your code is integrated with the style this file comes with.

Reply:
Ok noted!